AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -647,6 +647,7 @@ experts-exchange.com##.ctaSolutionStickyWrapper
 aliexpress.ru,aliexpress.com#$?##newuser-activity-config { remove: true; }
 !
 tempo.co##video[width="468"][height="60"]
+||goo.ne.jp/image_proxy/resize/w_600_h_200_q_100//topics/sports/s_banner$domain=news.goo.ne.jp
```

Comment:
Something like this `||img.news.goo.ne.jp/image_proxy/resize/*/topics/*/s_banner/` is ok without `$domain`.

Reply:
Done